lib/cli/src/cli.rs

```diff
@@ -204,5 +233,244 @@ pub fn wasmer_main() {
         }
     };
 
-    PrettyError::report(options.execute());
+    // Check if the file is a package name
+    if let WasmerCLIOptions::Run(r) = &options {
+        if !r.path.exists() {
+            let package = format!("{}", r.path.display());
+            if let Ok(mut sv) = split_version(&package) {
+                let mut package_download_info = None;
+                if !sv.package.contains('/') {
+                    let sp = spinner::SpinnerBuilder::new(format!(
+                        "Looking up command {} ...",
+                        sv.package
+                    ))
+                    .spinner(vec![
+                        "⣾", "⣽", "⣻", "⢿", "⡿", "⣟", "⣯", "⣷", " ", "⠁", "⠂", "⠄", "⡀", "⢀", "⠠",
+                        "⠐", "⠈",
```

Comment:
Since this is used/reused more times in the codebase, can we add it to a global var or isolate into a common function?